coqide.opam

```diff
@@ -20,7 +20,7 @@ depends: [
   "dune"               { build & >= "1.4.0" }
   "coqide-server"
   "conf-gtksourceview"
```

Comment:
Is this the right version?

Reply:
I did not see an opam package for lablgtk3 yet. I guess that an opam package `conf-gtksourceview3` or similar may have to be prepared at some time.

Reply:
There is one!
It is up to date with the released version, lablgtk-3.0.beta3.

Reply:
And there is a conf-gtksourceview3 too.

Reply:
Ah, ok!

Then, I upgraded the opam file (anticipating on the release number 3.0.0).